cachi2/core/models/sbom.py

```diff
@@ -93,3 +93,133 @@ class Sbom(pydantic.BaseModel):
     def _unique_components(cls, components: list[Component]) -> list[Component]:
         """Sort and de-duplicate components."""
         return unique_sorted(components, by=lambda component: component.key())
+
+    def to_spdx(self) -> "SPDXSbom":
+        """Convert a CycloneDX SBOM to an SPDX SBOM."""
+        return SPDXSbom(
+            packages=[
+                SPDXPackage(
+                    name=component.name,
+                    version=component.version,
+                    package_download_location=component.purl,
```

Comment:
What will replace `component.properties`? Those carry crucial information. Enterprise Contract inspects them and, for example, blocks builds if the SBOM has any `cachi2:pip:package:binary` properties

Reply:
I don't think there's anything equivalent for spdx package. See https://spdx.github.io/spdx-spec/v2.3/package-information/. Syft convert discards this information. I'll check with EC devs how to approach this

Reply:
A solution could be to keep using CycloneDX in cachi2 and not even add SPDX support. If a lossy conversion is all that Konflux needs, the build pipeline can use `syft convert` for that